shared/utilities/ticket-status/index.js

```diff
@@ -1,6 +1,37 @@
 import moment from 'moment';
 import { isBefore, isAfter, formatDate } from '../date';
 
+const defaultOptions = [
+  {
+    title: '',
+    ticketsAvailable: false,
+    waitingListOpen: false,
+    displayLevel: 'Featured',
```

Comment:
This data is used to select from the states below (see the one linked below), so you always get a valid message (though possibly not the right one if data is missing).

https://github.com/redbadger/react.london/blob/5968bddca1ad9b1cb09b3fd3b9b6bed5d2be350f/shared/utilities/ticket-status/index.js#L61